src/lib/viewers/BaseViewer.js

```diff
@@ -175,7 +175,9 @@ class BaseViewer extends EventEmitter {
         }
 
         const iconWrapperEl = container.querySelector(SELECTOR_BOX_PREVIEW_ICON);
-        iconWrapperEl.innerHTML = this.fileLoadingIcon || getIconFromName('FILE_DEFAULT');
+        if (iconWrapperEl) {
+            iconWrapperEl.innerHTML = this.fileLoadingIcon || getIconFromName('FILE_DEFAULT');
```

Comment:
Do we have a constant anywhere for `'FILE_DEFAULT'`?

Reply:
Nope, I can add it. We're not super consistent about what's defined as a constant vs what's not.